praw/objector.py

```diff
@@ -161,6 +162,23 @@ def objectify(
         if "json" in data and "errors" in data["json"]:
             errors = data["json"]["errors"]
             if len(errors) > 0:
+                for error in errors:
```

Comment:
I don't think this logic should live here, but we should catch RedditAPIException elsewhere.

Reply:
Should I move this code to the new objectify_request method?

Reply:
Yes, I think the APIException should be caught higher up and handled there.